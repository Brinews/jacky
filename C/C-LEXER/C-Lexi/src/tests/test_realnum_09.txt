
cse340.12 

  1.2<2.4
