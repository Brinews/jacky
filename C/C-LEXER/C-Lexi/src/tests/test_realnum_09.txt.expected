{cse340 , ID , 2}
{ , DOT , 2}
{12 , NUM , 2}
{1.2 , REALNUM , 4}
{ , LESS , 4}
{2.4 , REALNUM , 4}
{ , END_OF_FILE , 6}
